Source/Server/Cuelogic.Clrm.Api.Tests/Controllers/MasterGroupControllerTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Cuelogic.Clrm.Api.Controllers;
+using Cuelogic.Clrm.Service.Service;
```

Comment:
Why service contain service

Reply:
@nikhilb-cuelogic  - I have changes the folder hierarchy as per our discussion which accordingly changes the namespace